code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -92,6 +92,7 @@
 	var/secondary_langs = list()             // The names of secondary languages that are available to this species.
 	var/list/speech_sounds                   // A list of sounds to potentially play when speaking.
 	var/list/speech_chance                   // The likelihood of a speech sound playing.
+	var/accent                               // Special font species uses.
```

Comment:
I know, I'm being nitpicky, but you should probably mention the `loadedfonts` list in a comment here.